cypress/support/commands.js

```diff
@@ -10,7 +10,28 @@
 //
 //
 // -- This is a parent command --
-// Cypress.Commands.add("login", (email, password) => { ... })
+Cypress.Commands.add("login", (username, password) => {
+    cy.visit(Cypress.env('login_url'));
+        cy.get("[name=csrfmiddlewaretoken]")
+            .should("exist")
+            .should("have.attr", "value")
+            .as("csrfToken");
+
+        cy.get("@csrfToken").then((token) => {
+            cy.request({
+                method: "POST",
+                url: Cypress.env("login_url"),
+                form: true,
+                body: {
+                    username: username,
+                    password: password,
+                },
+                headers: {
+                    "X-CSRFTOKEN": token,
+                },
+            });
+        });
```

Comment:
This looks good, I am going to borrow it for a PR of mine in itwêwina tomorrow :)

Reply:
Awesome!

Reply:
I was accidentally omitting the password in the POST and the login was failing, so I ended up making sure that the login had actually worked:

```
          followRedirect: false
        }).then(response => {
          expect(response.status).to.eql(302)
          expect(response.headers).to.have.property('location')
          expect(response.headers.location).to.not.contain('login')
        })
```